apps/search/collections/google_search_results.coffee

```diff
@@ -1,27 +0,0 @@
-_ = require 'underscore'
-Backbone = require 'backbone'
-GoogleSearchResult = require '../models/google_search_result.coffee'
-{ GOOGLE_SEARCH_KEY, GOOGLE_SEARCH_CX } = require "../../../config"
-
-module.exports = class GoogleSearchResults extends Backbone.Collection
-  model: GoogleSearchResult
-
-  url: "https://www.googleapis.com/customsearch/v1?key=#{GOOGLE_SEARCH_KEY}&cx=#{GOOGLE_SEARCH_CX}"
-
-  parse:  (response) ->
-    _.reject response.items, (item) ->
-      # HACK filter out sensitive results
-      JSON.stringify(item).match(/kippenberger|zoe.*leonard|pat.*lipsky/i) or
-      # Filter out auction results
-      item.link?.indexOf('/auction-result') isnt -1 or
```

Comment:
So auctions (sales) are now fully supported? Or why was that filtered out before?